osu.Game.Rulesets.Diva/Objects/Drawables/DrawableDivaHitObject.cs

```diff
@@ -124,14 +128,13 @@ public override void PlaySamples()
 
         protected override void CheckForResult(bool userTriggered, double timeOffset)
         {
-            Console.WriteLine(timeOffset);
-            if (Judged || base.Time.Current < HitObject.StartTime)
```

Comment:
Here was the bug: `HitObject.StartTime` doesn't start when the object can be hit, it starts right at the middle -- when the object is set.
As such, this line set anything before the middle to fail.